Kitodo/src/main/java/org/kitodo/production/services/data/RulesetService.java

```diff
@@ -212,4 +215,12 @@ private QueryBuilder getRulesetsForCurrentUserQuery() {
                 ServiceManager.getUserService().getSessionClientId(), true));
         return query;
     }
+
+    public RulesetManagementInterface openRuleset(Ruleset ruleset) throws IOException {
```

Comment:
The method is used in `TitleRecordLinkTab`, line 248. It was taken from `DataEditorForm`. Instead of copy & paste, I have moved the common code to a place that seems reasonable to me and call the method from two places.